src/main/java/com/checkscanner/check/DataLoader.java

```diff
@@ -14,36 +13,27 @@
 import com.google.gson.Gson;
 import com.google.gson.GsonBuilder;
 
-
 @Component
-public class DataLoader
-{
-
+public class DataLoader {
 	@PostConstruct
-	public void loadSomeData()
-	{
+	public void loadSomeData() {
 		final File dir = new File("src/main/resources/venues/");
 		final File[] directoryListing = dir.listFiles();
 		final int count = 0;
 
-		if (directoryListing != null)
-		{
+		if (directoryListing != null) {
 
-			for (final File child : directoryListing)
-			{
-				try
-				{
+			for (final File child : directoryListing) {
+				try {
 					final Scanner scanner = new Scanner(child);
 					final String content = scanner.useDelimiter("\\Z").next();
 
 					final Gson gson = new GsonBuilder().create();
 					final Map venue = gson.fromJson(content, Map.class);
 
-					final List<Map> placeCategories = (List<Map>) venue.get("categories");
+					List<Map> placeCategories = (List<Map>) venue.get("categories");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "placeCategories" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1481)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this useless assignment to local variable "placeCategories". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1854)